code/code/disc/disc_psionics.cc

```diff
@@ -98,12 +93,21 @@ int TBeing::doPTell(const char* arg, bool visible) {
     sendTo("What a dumb master you have, charmed mobiles can't tell.\n\r");
     return FALSE;
   }
-  half_chop(arg, name, message);
 
-  if (!*name || !*message) {
+  char n[100];
+  char m[MAX_INPUT_LENGTH + 40];
+  half_chop(arg, n, m);
```

Comment:
Good call. I figured there was probably something in sstring that would do what half_chop was doing, but decided I'd rather refactor half-chop out of existence from everywhere all at once instead, in a future PR.

Reply:
And yeah, using single character variable names is dumb haha, but I figured since the scope of their use was so brief and it was so obvious what they were doing that in this case it was probably acceptable. Plus they'll be gone once half-chop isn't used any longer.